services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(
+            user=User.objects.get(username=username),
+        )
+        if date:
+            order.created_at = date
+        for ticket in tickets:
+            order.tickets.create(
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"],
+            )
+        order.save()
+        return order
+
+
+def get_orders(
+        username: str = None,
+) -> QuerySet:
```

Comment:
You could specify a `QuerySet` of what model instances you return in the return type annotation. Make this change here and in other views where it is applicable.